Clinical Key.js

```diff
@@ -0,0 +1,190 @@
+{
+	"translatorID": "a55463ba-e403-415b-80d4-284d5f9b4b15",
+	"label": "Clinical Key",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https://(www\\.|www-)clinicalkey(\\.|-)com",
```

Comment:
This is the url of my proxied connection: https://www-clinicalkey-com.ccmain.ohionet.org/#!